### DISCUSSION TITLE
Discussion link in menu needs to show number of discussions

### DISCUSSION THREAD

**Discussion by kcoyle** [community/community#3010] [Discussions]
Discussions looks empty compared to issues because there is no number beside the Discussions link in the header. Numbers would cause people to look to see what discussions are there.

**Comment by Dieforrobux** [comment]
Yeah.Good point it's pretty much blank. ' '.

**Comment by becca** [comment]
We're conversing about this over in https://github.com/github/feedback/discussions/2909#discussioncomment-181998
